src/term/term_to_net.rs

```diff
@@ -74,8 +74,17 @@ pub fn term_to_compat_net(term: &Term) -> (INet, u8) {
   // Encodes the main term.
   let mut global_vars = HashMap::new();
   let mut dups = 0;
-  let main =
-    encode_term(&mut inet, term, ROOT, &mut HashMap::new(), &mut vec![], &mut global_vars, &mut dups);
+  let mut tag_storage = HashMap::new();
```

Comment:
What do you think about storing the used dup tags in the Book during parsing?
I don't know if it wouldn't make it hard to compile individual terms.

Reply:
If you do, please rename `dups` to something more explicit like `untagged_dups`, `num_untagged_dups` or `untagged_dup_counter`.